[C#] Simplified basic-enums exercise by removing parsing (#864)
exercism/v3: languages/csharp/exercises/concept/enums/.docs/hints.md

### 1. Parse log level

> - There is a method to get a substring within a string, [Substring()](https://docs.microsoft.com/en-us/dotnet/api/system.string.substring?view=netframework-4.8)

Reviewer: ```suggestion
- There is a [method to get a part of a string](https://docs.microsoft.com/en-us/dotnet/api/system.string.substring?view=netcore-3.1).
```

Author: I really love this suggestion!

> - You can use a [switch-case statement](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/switch) to map each log level code to an enum.

Reviewer: ```suggestion
- You can use a [`switch` statement](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/switch) to elegantly handle the various log levels.
```

### 2. Support unknown log level

> - There's a [default](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/switch#the-default-case) statement to catch all unspecified cases.

Reviewer: ```suggestion
- There is a [special switch case](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/switch#the-default-case) that can be used to catch unspecified cases.
```
